src/consumption.cpp

```diff
@@ -1637,10 +1637,11 @@ time_duration Character::get_consume_time( const item &it )
                          5 ), 1 ) );  //Default 5 mL (1 tablespoon) per second
     float consume_time_modifier = 1;//only for food and drinks
     const bool eat_verb  = it.has_flag( flag_USE_EAT_VERB );
-    if( eat_verb || it.get_comestible()->comesttype == "FOOD" ) {
+    if( eat_verb || ( it.get_comestible() != nullptr && it.get_comestible()->comesttype == "FOOD" ) ) {
```

Comment:
Maybe worth factoring out the common code.  Have something like
```
const std::string comest_type = it.get_comestible() ? it.get_comestible()->comesttype : "";
```
before the `if`.  Might be cleaner code, or might not.  Just an idea.